addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -148,6 +148,16 @@ class UseSuggestion {
             before = text.substring(0, this.search.position);
             after = text.substring(position, text.length);
         }
+        if (option.isSpecial) {
+            if (option.type === "everyone") {
+                this.composer.mentionedPartners.add(...this.thread.channelMembers.map(
```

Comment:
`channelMembers` is local JS knowledge, all members might not be known/loaded at the time.

The best way would be to send "everyone" option to python and let it handle it instead, rather than trying to use standard JS params.